src/_normalise-reset.scss

```diff
@@ -660,6 +660,15 @@ progress {
 
 
     /**
+     * Allows textual input values to be positioned by padding alone.
+     */
+
+    #{$shell-g-textual-inputs} {
+        line-height: 1;
+    }
```

Comment:
This shouldn't be nested in the `.target-ios` selector. Also I always thought this as a **Base** thing in the **Forms** section: https://github.com/campaignmonitor/shell/blob/master/src/_base.scss#L112, what do you think?